### ISSUE TITLE
Improving the Separator Block - Gutenberg issues/PRs

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/twentytwentyone#278]
**Is your feature request related to a problem? Please describe.**
It would be nice to get improvements/options into the Separator Block as it could help improve the design.

![Screen Shot 2020-10-05 at 20 44 05](https://user-images.githubusercontent.com/5323259/95119001-8f1e4780-074b-11eb-9353-88c136aaaebb.png)

Existing Gutenberg issues/PR's:
**Separator Block: Add support for wide/full-width versions**
https://github.com/WordPress/gutenberg/issues/25080 -> https://github.com/WordPress/gutenberg/pull/25147
(The above has just been merged into Gutenberg.)

**Separator: allow changing the thickness / height**
https://github.com/WordPress/gutenberg/issues/20758 -> https://github.com/WordPress/gutenberg/pull/23403

**Separator: move color tools to the toolbar**
https://github.com/WordPress/gutenberg/issues/20757

**Blocks: Separator: Background style is too specific**
https://github.com/WordPress/gutenberg/issues/19539

**Add better variations for Separator block**
https://github.com/WordPress/gutenberg/issues/16483

Associated:
**Add Shape Divider block**
https://github.com/WordPress/gutenberg/issues/16351

**Comment by melchoyce** [comment]
![image](https://user-images.githubusercontent.com/2846578/95121713-6bdca380-071d-11eb-8038-455b8724c86a.png)

[Figma mockup](https://www.figma.com/file/OEtsCj8oD8JPVx4OvsS0U2/Twenty-Twenty-One?node-id=1394%3A0)

Unsure about the dots — right now it's just using the • • • characters. There's more potential for an interesting divider here. @ellenbauer do you have any ideas?

**Comment by paaljoachim** [comment]
I have made two new Separator block issues in the Gutenberg repo:
Adding styles such as waves, jagged and loops.
https://github.com/WordPress/gutenberg/issues/25991

Controlling the height of the Separator block
https://github.com/WordPress/gutenberg/issues/25989
(As one would not need to add as many spacer blocks if one can control the height of the Separator block.)

**Comment by carolinan** [comment]
With the pattern merged, are there any more actionable items here or can this be closed?

**Comment by paaljoachim** [comment]
Since I have made Gutenberg issues. Which I have listed. This issue might as well be closed.
